app/models/ministry.rb

```diff
@@ -28,8 +28,7 @@ class Ministry < ActiveRecord::Base
 
   has_many :user_content_locales, dependent: :destroy
   has_many :stories, dependent: :destroy
-  has_many :ministry_areas, dependent: :destroy
-  has_many :areas, through: :ministry_areas
+  belongs_to :area, dependent: :destroy
```

Comment:
Good catch and that's good to know about! I'll remove it. I think it got leftover from when I had it as `has_many` before.